metals/src/main/scala/scala/meta/internal/remotels/RemoteLanguageServer.scala

```diff
@@ -0,0 +1,114 @@
+package scala.meta.internal.remotels
+
+import org.eclipse.{lsp4j => l}
+import scala.meta.internal.metals.UserConfiguration
+import java.{util => ju}
+import scala.meta.io.AbsolutePath
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.JsonParser._
+import scala.util.Try
+import scala.meta.internal.metals.DefinitionResult
+import scala.meta.internal.semanticdb.Scala.Symbols
+import scala.meta.internal.metals.ReferencesResult
+import scala.meta.internal.metals.Buffers
+import scala.meta.internal.mtags.MD5
+import com.google.gson.JsonPrimitive
+import com.google.gson.JsonObject
+import org.eclipse.lsp4j.Location
+import scala.meta.internal.metals.BuildTargets
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext
+
+class RemoteLanguageServer(
+    workspace: () => AbsolutePath,
+    config: () => UserConfiguration,
+    buffers: Buffers,
+    buildTargets: BuildTargets
+)(implicit ec: ExecutionContext) {
+  def isEnabled: Boolean = config().remoteLanguageServer.isDefined
```

Comment:
I think this is only used in here and can probably be marked `private`?

Reply:
I inlined this method into `isEnabledForPath`